Imagine/Filter/Loader/RotateFilterLoader.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Liip\ImagineBundle\Imagine\Filter\Loader;
+
+use Imagine\Image\ImageInterface;
+use Imagine\Image\ManipulatorInterface;
+
+/**
+ * Loader for Imagine's basic rotate method
+ *
+ * @author Bocharsky Victor <bocharsky.bw@gmail.com>
+ */
+class RotateFilterLoader implements LoaderInterface
+{
+    /**
+     * Loads and applies a filter on the given image.
+     *
+     * @param ImageInterface $image
+     * @param array $options
+     *
+     * @return ManipulatorInterface
+     */
+    public function load(ImageInterface $image, array $options = array())
+    {
+        $angle = isset($options['angle']) ? intval(round(floatval($options['angle']), 0)) : 0;
```

Comment:
OK, I want to maintain a good precision in this place, but most probably it's unnecessary because angle in 0.5 degree will not be visible to the naked eye )